cvat-ui/src/reducers/annotation-reducer.ts

```diff
@@ -166,6 +166,8 @@ export default (state = defaultState, action: AnyAction): AnnotationState => {
                 activeShapeType = ShapeType.CUBOID;
             }
 
+            state.canvas.instance.destroy();
```

Comment:
Why not put it in the appropriate action?

Reply:
I've put it into `AnnotationActionTypes.DESTROY_CANVAS` action

Reply:
Did you comment these updates? Only annotation-reducer.ts was changed in this patch.

Reply:
From ``cvat-ui/``

Reply:
And should not we call destroy in BoundariesActionTypes.RESET_AFTER_ERROR action?

Reply:
I dont think we should call destroy in that action because in `annotation-reducer.ts` there are this kind of code:
```
case BoundariesActionTypes.RESET_AFTER_ERROR:
case AnnotationActionTypes.GET_JOB_SUCCESS: {
...
state.canvas.instance.destroy();
...
```
So probably there is no need to duplicate it in action

Reply:
Sure, got it.